crates/networking/p2p/discv4/server.rs

```diff
@@ -246,8 +240,7 @@ impl DiscoveryServer {
                 continue;
             }
 
-            if let Err(err) = self.send_find_node(&contact.node).await {
-                error!(sent = "FindNode", to = %format!("{:#x}", contact.node.public_key), err = ?err, "Error sending message");
```

Comment:
We should keep this as `debug` or `trace`

Reply:
I removed those because they were repeated log lines. They are being logged as `error` [here](https://github.com/lambdaclass/ethrex/blob/aada85a8cfd980efe85e6f52ce1af9d477c3cbde/crates/networking/p2p/discv4/server.rs#L520)